AndroidSDKCore/src/main/java/com/leanplum/internal/Constants.java

```diff
@@ -36,7 +36,7 @@
   public static int SOCKET_PORT = 80;
   public static int NETWORK_TIMEOUT_SECONDS = 10;
   public static int NETWORK_TIMEOUT_SECONDS_FOR_DOWNLOADS = 10;
-  public static String LEANPLUM_VERSION = "4.1.1";
+  public static String LEANPLUM_VERSION = BuildConfig.SDK_VERSION + '.' + BuildConfig.BUILD_NUMBER;
```

Comment:
Lets not concat these two things together, for example when we have a public release we dont want there to be any build number. Right now you have this defaulted to 0.

Reply:
I think we decided we _did_ want an _exact_ version number, which includes build. @e7mac was that right? This may have some downstream implications though if places assume "x.x.x" format. Do you know of any specific places, or places to check?